packages/devtools/src/parseErrorStack.ts

```diff
@@ -0,0 +1,52 @@
+import * as stacktraceParser from 'stacktrace-parser';
```

Comment:
Since this code is vendored from React Native repository, please also include the copyright header.